src/lib/api-endpoints.ts

```diff
@@ -65,16 +65,20 @@ export function searchResourcesEndpoint(
     startChapter?: number,
     endChapter?: number,
     startVerse?: number,
-    endVerse?: number
+    endVerse?: number,
+    offSet?: number,
+    limit?: number
 ): ApiStringAndCacheBustVersion {
     const resourceTypesParams = resourceTypes.map((rt) => `resourceTypes=${rt}`).join('&');
     return [
         `/resources/search?languageId=${languageId}${query ? `&query=${query}` : ''}${
```

Comment:
This seems to be getting a bit unwieldy, would you be open to building a `new URLSearchParams()` and then setting each item and doing a `toString()` on it?